--- /home/pi/pi-gen/stage3/00-install-packages/00-packages	2025-10-22 19:13:16.760120709 -0700
+++ /home/pi/pi-gen-cstm/stage3/00-install-packages/00-packages	2025-10-20 20:14:21.899017538 -0700
@@ -1,2 +1,21 @@
-rpd-preferences
-rpd-theme
+##rpd-preferences
+rpcc
+pipanel
+rc-gui
+#raindrop
+rasputin
+#rpinters
+#rp-prefapps
+#cups
+#cups-pk-helper
+#pi-printer-support
+
+##rpd-theme
+#pixtrix-theme
+#pixtrix-icons
+#fonts-nunito-sans
+#fonts-liberation2
+rpd-wallpaper
+rpd-wallpaper-trixie
+
+xdg-user-dirs
\ No newline at end of file
