--- /home/pi/pi-gen/stage3/00-install-packages/00-packages-nr
+++ /home/pi/pi-gen-cstm/stage3/00-install-packages/00-packages-nr
@@ -1,2 +1,26 @@
-rpd-wayland-core
-rpd-x-core
+##rpd-wayland-core
+##rpd-x-core
+i3 i3-wm i3status
+i3lock
+dunst
+feh
+lxappearance
+lxterminal
+#mesa-vulkan-drivers
+network-manager-gnome
+picom
+#poppler-data
+picom
+rofi
+x11-common
+x11-utils
+xdg-utils
+#xfonts-encodings
+#xfonts-utils
+xinit
+xserver-common
+xserver-xorg
+xserver-xorg-core
+xserver-xorg-input-libinput
+xserver-xorg-video-fbdev
+xss-lock
