--- /home/pi/pi-gen/stage3/00-install-packages/00-packages-nr	2025-10-22 19:13:16.760120709 -0700
+++ /home/pi/pi-gen-cstm/stage3/00-install-packages/00-packages-nr	2025-10-20 20:13:49.582994366 -0700
@@ -1,2 +1,29 @@
-rpd-wayland-core
-rpd-x-core
+##rpd-wayland-core
+
+##rpd-x-core
+i3
+dunst
+fonts-urw-base35
+libflac14
+libogg0
+libvorbis0a
+lxterminal
+poppler-data
+rofi
+x11-common
+x11-xkb-utils
+xdg-utils
+xfonts-encodings
+xfonts-utils
+xinit
+xserver-common
+xserver-xorg
+xserver-xorg-core
+xserver-xorg-input-all
+xserver-xorg-input-libinput
+#xserver-xorg-video-all
+#xserver-xorg-video-amdgpu
+#xserver-xorg-video-ati
+xserver-xorg-video-fbdev
+#xserver-xorg-video-nouveau
+#xserver-xorg-video-radeon
