--- /home/pi/pi-gen/stage0/00-configure-apt/00-run.sh	2025-10-19 18:56:08.498847151 -0700
+++ /home/pi/pi-gen-cstm/stage0/00-configure-apt/00-run.sh	2025-10-19 18:14:44.259623046 -0700
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 
 true > "${ROOTFS_DIR}/etc/apt/sources.list"
+install -m 644 files/80suggests "${ROOTFS_DIR}/etc/apt/apt.conf.d/"
 install -m 644 files/debian.sources "${ROOTFS_DIR}/etc/apt/sources.list.d/"
 install -m 644 files/raspi.sources "${ROOTFS_DIR}/etc/apt/sources.list.d/"
 sed -i "s/RELEASE/${RELEASE}/g" "${ROOTFS_DIR}/etc/apt/sources.list.d/debian.sources"
