--- /home/pi/pi-gen/stage2/02-net-tweaks/00-packages	2025-09-06 22:42:03.887710268 -0700
+++ /home/pi/pi-gen-cstm/stage2/02-net-tweaks/00-packages	2025-09-06 22:41:26.047596836 -0700
@@ -1,4 +1,8 @@
-wpasupplicant wireless-tools firmware-atheros firmware-brcm80211 firmware-libertas firmware-realtek firmware-mediatek firmware-marvell-prestera-
+wpasupplicant wireless-tools firmware-brcm80211 firmware-realtek
 raspberrypi-net-mods
 network-manager
 net-tools
+
+dnsmasq
+dns-root-data
+wireless-regdb
\ No newline at end of file
