--- /home/pi/pi-gen/stage2/02-net-tweaks/00-packages	2025-09-02 17:50:09.009568936 -0700
+++ /home/pi/pi-gen-cstm/stage2/02-net-tweaks/00-packages	2025-09-02 17:53:55.777763997 -0700
@@ -1,4 +1,4 @@
-wpasupplicant wireless-tools firmware-atheros firmware-brcm80211 firmware-libertas firmware-realtek firmware-mediatek firmware-marvell-prestera-
+wpasupplicant wireless-tools firmware-brcm80211 firmware-realtek
 raspberrypi-net-mods
 network-manager
 net-tools
