--- /home/pi/pi-gen/stage2/03-set-timezone/02-run.sh	2025-08-19 20:21:20.708048758 -0700
+++ /home/pi/pi-gen-cstm/stage2/03-set-timezone/02-run.sh	2025-08-19 20:49:20.116453585 -0700
@@ -3,6 +3,8 @@
 echo "${TIMEZONE_DEFAULT}" > "${ROOTFS_DIR}/etc/timezone"
 rm "${ROOTFS_DIR}/etc/localtime"
 
+sed -i "s/^#NTP=/NTP=us.pool.ntp.org/" "${ROOTFS_DIR}/etc/systemd/timesyncd.conf"
+
 on_chroot << EOF
 dpkg-reconfigure -f noninteractive tzdata
 EOF
