Index: stage2/rootfs/etc/inputrc
===================================================================
--- stage2.orig/rootfs/etc/inputrc
+++ stage2/rootfs/etc/inputrc
@@ -22,8 +22,8 @@
 # set bell-style visible
 
 # make autocompletion case insensitive and display suggestions
-# set completion-ignore-case On
-# set show-all-if-ambiguous On
+set completion-ignore-case On
+set show-all-if-ambiguous On
 
 # some defaults / modifications for the emacs mode
 $if mode=emacs
@@ -65,7 +65,7 @@
 # "\eOF": end-of-line
 
 # for freebsd console
-# "\e[H": beginning-of-line
-# "\e[F": end-of-line
+"\e[H": beginning-of-line
+"\e[F": end-of-line
 
 $endif
