--- /home/pi/pi-gen/stage2/01-sys-tweaks/00-packages	2025-08-05 21:25:18.273025848 -0700
+++ /home/pi/pi-gen-cstm/stage2/01-sys-tweaks/00-packages	2025-08-05 21:20:22.245009530 -0700
@@ -1,25 +1,32 @@
-ssh less fbset sudo psmisc strace ed ncdu
-console-setup keyboard-configuration debconf-utils parted
-build-essential manpages-dev bash-completion gdb pkg-config
+ssh less 
+#fbset 
+sudo psmisc 
+#strace ed ncdu
+console-setup keyboard-configuration 
+#debconf-utils 
+parted
+#build-essential manpages-dev pkg-config
+bash-completion gdb
 python-is-python3
-v4l-utils
-gpiod python3-libgpiod
-python3-gpiozero
-pigpio python3-pigpio raspi-gpio python3-rpi-lgpio
-python3-spidev
-python3-smbus2
+#v4l-utils
+#gpiod python3-libgpiod
+#python3-gpiozero
+#pigpio python3-pigpio raspi-gpio python3-rpi-lgpio
+#python3-spidev
+#python3-smbus2
 avahi-daemon
 lua5.1
 luajit
 ca-certificates curl
-fake-hwclock nfs-common usbutils
-dosfstools
+#fake-hwclock nfs-common 
+usbutils
+dosfstools exfatprogs
 dphys-swapfile
 raspberrypi-sys-mods
-pi-bluetooth
-apt-listchanges
+#pi-bluetooth
+#apt-listchanges
 usb-modeswitch
-libpam-chksshpwd
+#libpam-chksshpwd
 rpi-update
 libmtp-runtime
 rsync
@@ -28,12 +35,23 @@
 policykit-1
 ssh-import-id
 ethtool
-ntfs-3g
+#ntfs-3g
 pciutils
 rpi-eeprom
 raspi-utils
 udisks2
-unzip zip p7zip-full
+unzip zip 
+#p7zip-full 
+xz-utils zstd
 file
 kms++-utils
 python3-venv
+
+avahi-autoipd
+gawk
+lshw
+manpages
+needrestart
+patch
+python3-toml
+userconf-pi dos2unix
